src/lib.rs

```diff
@@ -0,0 +1,41 @@
+// use crate::database::driver::DatabaseDriver;
+
+use config::configuration::Config;
+use validator::Validate;
+
+use crate::database::sqlite;
+
+pub mod cli;
+pub mod config;
+pub mod database;
+pub mod f2b;
+pub mod logging;
+
+/// Public interface for initializing logging.
+fn init_logging(logging_environment: String) {
+    logging::set_logging_environment(logging_environment);
+    env_logger::init();
+    log::info!("Starting fail2ban-monitoring");
+}
+
+fn parse_configuration() -> Config {
+    log::info!("Reading configuration file");
+    config::configuration::Config::new().read_configuration()
+}
```

Comment:
I'm not 100% sure about these 2 methods existing just for 1 or 2 lines of actual code, but maybe you have more things in mind!